_episodes/08-ilamb-exercise-1.md

```diff
@@ -753,6 +738,28 @@ qsub ilamb_test.job
 
 Running this job will create a `_build` directory with the comparison results within `$ILAMB_ROOT`. You can adjust the place of this directory via a agrument `--build_dir` argument for `ilamb-run`.
 
+## View Result
+
+Once you finish your `ILAMB` run, you will get your `ILAMB` result. The default path to the result is `./_built`，unless you specified `--build_dir` before you run your experiment with `ILAMB`.
+
+Then you change your directory to this path, create a new local host by below command:
+```bash
+python3 -m http.server
+```
+Your `ILAMB` result can be viewed in the following address: localhost address:
+```
+http://0.0.0.0:8000/
+```
+![image](../fig/ilamb_result-matrix.png)
```

Comment:
Not sure this is the best way to do. We will use VSCode for ESMValTool... Might be good to do the same here. 
@rhaegar325 ?